portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Rebeka Dittrich</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="socialnav">
+      <a href="https://uk.linkedin.com" target="_blank">LinkedIn<a>
+      <a href="https://github.com/rebekadittrich" target="_blank">GitHub</a>
+    </div>
+    <div class="row">
+      <div class="column">
+        <div class="mainnav">
+          <a href="index.html">About me</a>
+          <a href="projects.html">Resume</a>
+          <a class="outline" href="gallery.html">Gallery</a>
+        </div>
+      </div>
+      <div class="column">
+        <div id="content">
+          <h1>Gallery</h1>
+          <p>Few of my paintings and drawings.</p>
+          <img src="/images/gallery/g3.jpg">
+          <img src="/images/gallery/g2.JPG">
+          <img src="/images/gallery/g1.jpg">
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
Please add a new line at the end of every file. That is just for being consistent that a line in a file always ends with a newline char (tooling maybe depends on this).